Makefile

```diff
@@ -32,10 +32,10 @@ clean-tests:
 	KUBECONFIG=$(KUBECONFIG) kubectl delete persistentvolumeclaims --all
 
 test-driver:
-	go test -v github.com/UpCloudLtd/upcloud-csi/driver
+	go test -race -v github.com/UpCloudLtd/upcloud-csi/driver
```

Comment:
Should we run tests against all the packages, smth like:
`go test -race -v ./...`

Reply:
@i-prudnikov thank you for comments. Good point! We had some broken tests at some point but I think that now those are skipped and we can change that make target already to test all packages. I'll push changes.